frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -209,33 +208,6 @@ class Sidebar extends Component<Props> {
               </NavLink>
             </div>
           ) : null }
-          { deviceIDs.map(deviceID => (
-            <div key={deviceID} className={style.sidebarItem}>
-              <NavLink
-                to={`/device/${deviceID}`}
-                className={({ isActive }) => isActive ? style.sidebarActive : ''}
-                title={t('sidebar.device')}
-                onClick={this.handleSidebarItemClick}>
-                <div className={style.single}>
-                  <img draggable={false} src={deviceSettings} alt={t('sidebar.device')} />
-                </div>
-                <span className={style.sidebarLabel}>{t('sidebar.device')}</span>
-              </NavLink>
-            </div>
-          )) }
-          <div key="settings" className={style.sidebarItem}>
-            <NavLink
-              className={({ isActive }) => isActive ? style.sidebarActive : ''}
-              to={'/settings'}
-              title={t('sidebar.settings')}
-              onClick={this.handleSidebarItemClick}>
-              <div className="stacked">
-                <img draggable={false} src={settingsGrey} alt={t('sidebar.settings')} />
-                <img draggable={false} src={settings} alt={t('sidebar.settings')} />
-              </div>
-              <span className={style.sidebarLabel}>{t('sidebar.settings')}</span>
-            </NavLink>
-          </div>
```

Comment:
Removed "Manage Device" and "Settings" (the old one) from the sidebar :) 

For BB01, the manage device will also be inside the new settings. Only once the BB01 user clicks "Manage device", they'll be brought to the old (current) manage device of BB01, which I believe is here: 
https://github.com/digitalbitbox/bitbox-wallet-app/blob/0995f0a550384cbed44b8b15e0459a765b152c23/frontends/web/src/routes/device/bitbox01/settings/settings.jsx#L40